cypress_test/integration_tests/common/desktop_helper.js

```diff
@@ -68,15 +68,14 @@ function showStatusBarIfHidden() {
 function showSidebarIfHidden() {
 	cy.log('>> showSidebarIfHidden - start');
 
-	cy.get('#sidebar')
-		.then(function(sidebarItem) {
-			if (!sidebarItem.hasClass('checked')) {
-				showSidebar();
-			}
-		});
+	cy.get('#sidebar-dock-wrapper')
```

Comment:
I guess this conflicts with the "ifHidden" part of the function. This condition forces it to be hidden. If we use this that way, we need to make sure that the caller functions are sure that the sidebar is hidden. Same goes for below changes it seems.

Reply:
Exactly, in cypress we shouldn't use "ifs" - as a rule of thumb

Test should have strict checks - that makes it a lot easier to debug and reproduce.
Then we have no races caused by temporary states.

Also `then` is not retried, it should not be used for "checks" but rather "modifications".
https://docs.cypress.io/api/commands/should#Whats-the-difference-between-then-and-shouldand

Reply:
I don't think "then" here would cause anything. Since it's not trying anything. cy.get is doing that.

And that should be "cy.cGet" i believe, not "cy.get"?

Anyway, this removes the "ifHidden" part. So there are 3 things here:
* Function name should be changed to "showSidebar".
* Caller functions should be sure that the sidebar is hidden.
* _We already have a "showSidebar" function_. Maybe we can remove this function entirely and redirect the calls to "showSidebar" function in this case.